examples/capture/main.c

```diff
@@ -33,7 +33,7 @@ int main(
                 },
                 .maxBindGroups = 1,
                 .label = "Device",
-                .tracePath = NULL,
+                .tracePath = "build/",
```

Comment:
leftover?

Reply:
was making it match the other one, maybe that was leftover? https://github.com/gfx-rs/wgpu-native/blob/master/examples/compute/main.c#L35